tests/test_series.py

```diff
@@ -417,19 +417,16 @@ def test_types_sort_values_with_key() -> None:
 
 def test_types_shift() -> None:
     s = pd.Series([1, 2, 3], index=pd.date_range("2020", periods=3))
-    # Return type "Series[int]"" not quite correct
-    # - https://github.com/pandas-dev/pandas-stubs/issues/1111
-    # - https://github.com/pandas-dev/pandas-stubs/issues/1110
-    check(assert_type(s.shift(), "pd.Series[int]"), pd.Series, np.floating)
+    check(assert_type(s.shift(), "pd.Series[Any]"), pd.Series, np.floating)
```

Comment:
if you change the PYI file to not reference `Any`, in the tests, you just do `assert_type(s.shift(), pd.Series)` without the quotes